docs/usage/configuration/config_documentation.md

```diff
@@ -3670,14 +3671,63 @@ opentracing:
         false
 ```
 ---
-## Workers ##
+## Worker configuration for main process
 Configuration options related to workers.
+Workers are used to scale horizontally and distribute the load to different processes.
+
+The worker configuration is divided into two parts.
+
+1. These switches for the shared configuration to give the main process
+   the necessary information.
+1. And the [switches for the worker configuration file](#worker-configuration-for-workers-configuration)
+   to configure each worker them self are futher below.
+
+The manual how to configure workers is described in detail in the
+[worker documentation](../../workers.md)
+
+---
+### `worker_replication_secret`
+
+A shared secret used by the replication APIs to authenticate HTTP requests
+from workers.
+
+By default this is unused and traffic is not authenticated.
+
+Example configuration:
+```yaml
+worker_replication_secret: "secret_secret"
+```
+---
+### `start_pushers`
+
+Controls sending of push notifications on the main process. Set to `false`
+if using a [pusher worker](../../workers.md#synapseapppusher). Defaults to `true`.
+
+Example configuration:
+```yaml
+start_pushers: false
+```
+---
+### `pusher_instances`
+
+It is possible to run multiple [pusher workers](../../workers.md#synapseapppusher),
+in which case the work is balanced across them. Use this setting to list the pushers by
+[`worker_name`](#worker_name).
+
+If only one pusher worker is configured, this setting is not necessary.
 
+Example configuration:
+```yaml
+pusher_instances:
+  - pusher_worker1
+  - pusher_worker2
```

Comment:
Good idea. I have added it.

Reply:
It's unnecessary. See comment above.

Reply:
I don't have strong opinions other than: I would rather we deprecate `start_pushers` rather than deal with having two related options interact in a confusing way. But that's not something to deal with here.